### DISCUSSION TITLE
'ls' is not recognized as an internal or external command, operable program or batch file.

### DISCUSSION THREAD

**Discussion by malam95** [Eugeny/tabby#8188] [Q&A]
in default tabby 'ls' is not recognized as an internal or external command,
operable program or batch file. , what can i do

**Comment by lawweiliang** [comment]
Hi @malam95,
What do you mean? Did you mean Tabby not recognized 'ls' command?

Do take `notes`.
 `ls' command is from GNU/Linux operating system. Do not mix it with tabby.
 Tabby is just terminal software. 
 
 ---
In order for you to use `ls` command with tabby in window environment.

There are three ways.
1. Vmware/VirtualBox + Linux + Tabby
2. Git Bash + Tabby
3. WSL2 + Linux + Tabby

Personally, I prefer method 1.

---
